File: Soal A/Hardware.java
import javax.swing.plaf.metal.MetalBorders.PopupMenuBorder;

public class Hardware extends Product{

    private String brand = "";
    private String model = "";

    public Hardware(String brand, String model) {
        this.brand = brand;
        this.model = model;
    }

    public Hardware() {

    }

    public void setBrand(String brand) {
        this.brand = brand;
    }
    public String getBrand() {
        return this.brand;
    }

    public void setModel(String model) {
        this.model = model;
    }
    public String getModel() {
        return this.model;
    }

}

File: Python/Hardware.py
from Product import Product

class Hardware(Product):
    
    # constructor
    def __init__(self):
        self.brand = "blank"
        self.model = "blank"
        
    # methods for brand attributes
    def setBrand(self, brand):
        self.brand = brand
        
    def getBrand(self):
        return self.brand 

    # methods for model attributes
    def setModel(self, model):
        self.model = model
        
    def getModel(self):
        return self.model 
